galsim/real.py

```diff
@@ -1294,8 +1294,13 @@ def _initialize(self, imgs, bands, xis, PSFs,
             NSED, Nim, nk, nk)
 
         # Reorder these so they correspond to (NSED, nky, nkx) and (NSED, NSED, nky, nkx) shapes.
-        coef = np.moveaxis(coef, (0,1,2), (1,2,0))
-        Sigma = np.moveaxis(Sigma, (0,1,2,3), (2,3,0,1))
+        try:
```

Comment:
Thanks.  That's better.  I didn't know about the version of transpose that takes a list of ints.

Although, the convention is different, so the first one is actually (2,1,0), not (1,2,0).  And of course the [example](https://docs.scipy.org/doc/numpy/reference/generated/numpy.transpose.html) that numpy gives in the documentation is ambiguous about which way the permutation happens.  :-P